tests/codeception/acceptance/users_frontend.feature

```diff
@@ -45,6 +46,7 @@ Feature: Users Frontend
     Then I should see the name "User Three"
 
   Scenario: Test last login date
-    Given Needs to user "User Two" logged in at least once
-    When I login as a super admin from backend
+    Given I am on the User Manager page
+    When I Needs to user "User Two" logged in at least once
+    And I login as a super admin from backend
```

Comment:
`And I login as a super admin from backend` is already covered by `Given I am on the User Manager page`, isn't it?

```Gherkin
Given  I am on the User Manager page
And    User :arg1 did login at least once
```